src/main/java/ch/njol/skript/test/runner/CondRunningJUnit.java

```diff
@@ -0,0 +1,60 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.test.runner;
+
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.NoDoc;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Check JUnit")
+@Description({
+	"Returns true if the test runner is currently running a JUnit.",
+	"Useful for the EvtTestCase of JUnit exclusive syntaxes registered from within the test packages."
+})
+@NoDoc
+public class CondRunningJUnit extends Condition {
+
+	static {
+		Skript.registerCondition(CondRunningJUnit.class, "running JUnit");
```

Comment:
we once had an issue with UUID due to capitalization but fixed, I think we should just stick with lowercase syntax.
```suggestion
		Skript.registerCondition(CondRunningJUnit.class, "running junit");
```